runtime/doc/repeat.txt

```diff
@@ -253,21 +253,25 @@ For writing a Vim script, see chapter 41 of the user manual |usr_41.txt|.
 			below "plugin", just like with plugins in
 			'runtimepath'.
 
-			If the filetype detection was not enabled yet (this
+			If the filetype detection was already enabled (this
 			is usually done with a "syntax enable" or "filetype
-			on" command in your .vimrc file), this will also look
+			on" command in your .vimrc file, or automatically during
```

Comment:
Since we're making Neovim specific changes, might as well go all in.

```suggestion
			on" command in your |init.vim|, or automatically during
```